doc-content/shared-kie-docs/src/main/asciidoc/KieServer/KieServer-chapter.adoc

```diff
@@ -29,7 +29,7 @@ include::controller-java-api-requests-proc.adoc[leveloffset=+2]
 include::controller-java-clients-ref.adoc[leveloffset=+2]
 include::controller-java-api-examples-ref.adoc[leveloffset=+2]
 
-include::SecuringPasswords.adoc[leveloffset=+1]
+include::securing-passwords-proc.adoc[leveloffset=+1]
```

Comment:
If this new proc file replaces the old one, go ahead and delete the old `SecuringPasswords.adoc` file to keep things tidy. Just be sure to search the file name and anchor ID to address anywhere else it's either included or cross-referenced.

Also, your community doc previews aren't quite right. The "jbpm" version is actually the Drools doc, and your "drools" version is 404ing. Also, I found a cross-reference to the old securing passwords section in this section: http://file.ork.redhat.com/~emmurphy/BXMSDOC-4486-jbpm/#_wb.systemproperties  In the Drools output, search "securing passwords" and you'll see it. I don't think it's your fault because the securing section didn't even have an anchor ID, and this cross references was created using the default ascii-generated anchor ID from the output itself. I've seen this a few times before. The only way around it is to search the old title name in the output after you make the change to confirm any lingering refs like that.